src/lib/errors/base-http-error.js

```diff
@@ -1,46 +1,46 @@
-function BaseHttpError(message, code) {
-  const defaultMessage = 'Server-side error';
-
-  // HTTP status code
-  this.code = code || 500;
-  // Main full message
-  this.message = this.formatMessage(message || defaultMessage);
-  // Full stack of messages
-  this.messages = [this.message];
-
-  Error.captureStackTrace(this, this.constructor);
+// Stub for extends from error in ES6 classes.
+// NOTE/TODO: Remove it and do extends from Error when this behavior will be fixed.
+function ErrorStub(message) {
```

Comment:
Do it like temporally thing for more clarify syntax. I hope this not bad way. :)